src/Perlang.ConsoleApp/Program.cs

```diff
@@ -487,6 +517,7 @@ public string[] GetSuggestions(string text, int index)
             {
                 var matchingKeywords = Scanner.ReservedKeywords
                     .Where(keyword => keyword.Key.StartsWith(text))
+                    .Where(keyword => keyword.Value != TokenType.RESERVED_WORD)
```

Comment:
(Drive-by-fix: the reserved words doesn't make any sense to include in the `GetSuggestions` reuslt, since they are just "reserved for future use" in that sense.)